typescript/cli/src/config/ism.ts

```diff
@@ -73,6 +73,7 @@ const ISM_TYPE_DESCRIPTIONS: Record<IsmType, string> = {
     'ISM where you can deliver messages without any validation (WARNING: only for testing, do not use in production)',
   [IsmType.OP_STACK]: '',
   [IsmType.PAUSABLE]: '',
+  [IsmType.CUSTOM]: '',
```

Comment:
let's only add these if they're going to be implemented per the recent bug bash

see https://github.com/hyperlane-xyz/hyperlane-monorepo/pull/3908

Reply:
fixed